html/changelogs/KingOfThePing - 13789.yml

```diff
@@ -0,0 +1,5 @@
+author: KingOfThePing
```

Comment:
```suggestion
author: RyverStyx, KingOfThePing
```

Reply:
I guess double crediting is the way to go.